.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "printWidth": 84,
+  "tabWidth": 2,
+  "useTabs": false,
+  "semi": true,
+  "singleQuote": true,
+  "trailingComma": "none"
+}
```

Comment:
Let's stick to Prettier defaults (eventually we may put `printWidth` to `100`).

Also let's put this config into `package.json`

Reply:
Done